packages/ui-extensions/src/surfaces/admin/components/LocationList/LocationList.doc.ts

```diff
@@ -0,0 +1,41 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
```

Comment:
As we're not intended to show this components to developers we should remove the docs 

@olavoasantos do we have a way to test that the docs will be correctly built if we don't add any files? 

Or maybe preserve the `doc` but use any config to hide it?

Reply:
Done, @MitchLillie! 🙏